features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/db2.sql

```diff
@@ -2699,6 +2699,8 @@ CREATE TABLE AM_KEY_MANAGER (
   CONFIGURATION BLOB NULL,
   ENABLED SMALLINT DEFAULT 1,
   TENANT_DOMAIN VARCHAR(100) NOT NULL,
```

Comment:
We need to rename this column and the required queries. We can do this in a separate round.
@wasuradananjith @Nirothipan please note

Reply:
Created a separate issue [1] to separately track this. Thanks

[1] https://github.com/wso2-enterprise/choreo/issues/5149